pkg/kp/rcstore/consul_store.go

```diff
@@ -220,6 +220,16 @@ func (s *consulStore) SetDesiredReplicas(id fields.ID, n int) error {
 	})
 }
 
+func (s *consulStore) MutateDesiredReplicas(id fields.ID, n int) error {
```

Comment:
yeah that's a good change. anyone else have opinions on this name? otherwise I'll use AddDesiredReplicas instead